src/main/resources/application-local.yaml

```diff
@@ -11,3 +11,6 @@ network-store-server:
 actions-server:
   base-uri: http://localhost:5022
 
+# default values for loadflow providers "Hades2" or "OpenSecurityAnalysis"
+loadflow:
+  default-provider: OpenSecurityAnalysis
```

Comment:
Missing newline

Reply:
property moved to application.yaml, with new line added